### ISSUE TITLE
Cannot install with yarn in combination with NodeJS 9

### ISSUE THREAD

**Issue by ofnof** [nickcolley/jest-axe#12]
When installing jest-axe with the command `yarn add jest-axe -D` on a NodeJS 9 system the following error appears:

```
error jest-axe@2.1.1: The engine "node" is incompatible with this module. Expected version "8.9.4".
error Found incompatible module
```

If i directly require a copy of the *jest-axe/index.js* file in my project then all is well. So removing/changing the *engines/node* variable in *package.json* might resolve this problem.

**Comment by nickcolley** [comment]
Thanks for raising this, will see if I can .Ake the engines less strict!

**Comment by nickcolley** [comment]
Released as 2.1.2 thanks @ofnof for raising and @connor-baer for fixing :)
